nc-news-controllers/nc-news.controller.js

```diff
@@ -0,0 +1,7 @@
+const {selectTopics} = require('../models/nc-news.model.js');
+
+exports.getTopics = (req, res, next) => {
+  selectTopics().then((topics) => {
+    res.status(200).send({topics});
+  });
```

Comment:
I'd add a catch block with `next` to this just to account for the unexpected. Nothing else to test, just a precaution.